dp/generate_climos.py

```diff
@@ -169,11 +184,9 @@ def climo_outputs(time_resolution):
         cf.climo_start_time = t_start.isoformat()[:19] + 'Z'
         cf.climo_end_time = t_end.isoformat()[:19] + 'Z'
         if hasattr(input_file, 'tracking_id'):
-            cf.climo_tracking_id=input_file.tracking_id
-        try:
-            cf.variables['pr'].units = pr_units_attr
-        except NameError:
```

Comment:
An answer in two parts:

1. The original try-catch block was just a shitty way decide if there were any updates to be done. Much better now.
2. If the variable is not actually attributed with units, the code will raise an AttributeError exception from netcdf. But we don't yet have an nclint check that all variables are attributed with units. (And we could probably add some other attributes to check as well.)